data-serving/data-service/test/model/demographics.test.ts

```diff
@@ -0,0 +1,62 @@
+import {
+    DemographicsDocument,
+    demographicsSchema,
+} from '../../src/model/demographics';
+
+import { Error } from 'mongoose';
+import mongoose from 'mongoose';
+
+const Demographics = mongoose.model<DemographicsDocument>(
+    'Demographics',
+    demographicsSchema,
+);
+
+describe('validate', () => {
+    it('a start age under -1 is invalid', async () => {
+        return new Demographics({ ageRange: { start: -1.1 } }).validate((e) => {
+            expect(e.name).toBe(Error.ValidationError.name);
+        });
+    });
+    it('a start age over 300 is invalid', async () => {
```

Comment:
The limit is 200, though I think it'd be reasonable just to pick some obviously-too-large age rather than focus on the exact cut-off.

Reply:
oops, the limit should be 300. was supposed to update this to be in sync with the schema definition (keeping these in sync is hard)

300 was Tim's suggestion to support other species